src/components/tooltip/styles.ts

```diff
@@ -0,0 +1,132 @@
+import { styled } from '@linaria/react'
+
+export const ElToolTipContainer = styled.div`
+  position: relative;
+  display: inline-block;
+  overflow: visible;
+`
+
+export const ElToolTipChild = styled.div`
```

Comment:
**suggestion:** I think this should be `ElTooltip` since this represents the actual presentational "tooltip" atom.